lambda/src/data.rs

```diff
@@ -8,4 +8,8 @@ pub const STREAMS: phf::Map<&'static str, &'static str> = phf_map! {
     "knuth" => "https://www.youtube.com/watch?v=10XjXGPzAyE",
     "spencer" => "https://www.youtube.com/watch?v=zHW5F4EthHU",
     "brattain" => "https://www.youtube.com/watch?v=pm6gNIm4dnU",
+    "feynman" => "https://youtube.com/live/2QSo1Zb2Xo4?feature=share",
+    "tesla" => "https://youtube.com/live/3ppW3ZNnmDA?feature=share",
+    "hawking" => "https://youtube.com/live/Me98sMD5KBs?feature=share",
+    "pasteur" => "https://youtube.com/live/pzwmqGg6mT4?feature=share"
```

Comment:
nit (non-blocking, opinionated): IMO we should leave a trailing comma for convenience reordering/editing lines for whatever reason. (but also, there's, like, no linter here so 🤷🏻 )

Reply:
nit: also just noticed the `?feature=share`. doesn't matter, but also \*ew*